database/migrations/2019_07_12_045144_create_plans_table.php

```diff
@@ -0,0 +1,92 @@
+                $table->softDeletes();
+                $table->timestamp('created_at')->default(DB::raw('CURRENT_TIMESTAMP'));
+                $table->timestamp('updated_at')->default(DB::raw('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'));
+            });
+        }
+
+        if (!Schema::connection('dbp_users')->hasTable('plan_days')) {
+            Schema::connection('dbp_users')->create('plan_days', function (Blueprint $table) {
+                $table->bigIncrements('id');
+                $table->bigInteger('plan_id')->unsigned();
+                $table->foreign('plan_id', 'FK_plan_days')->references('id')->on(config('database.connections.dbp_users.database') . '.plans')->onDelete('cascade')->onUpdate('cascade');
+                $table->timestamp('created_at')->default(DB::raw('CURRENT_TIMESTAMP'));
+                $table->timestamp('updated_at')->default(DB::raw('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'));
+                $table->integer('order_column');
+            });
+        }
+
+        if (!Schema::connection('dbp_users')->hasTable('plan_playlist_items')) {
+            Schema::connection('dbp_users')->create('plan_playlist_items', function (Blueprint $table) {
+                $table->bigIncrements('id');
+                $table->bigInteger('day_id')->unsigned()->nullable();
+                $table->foreign('day_id', 'FK_plan_days_items')->references('id')->on(config('database.connections.dbp_users.database') . '.plan_days')->onDelete('cascade')->onUpdate('cascade');
+                $table->bigInteger('playlist_id')->unsigned()->nullable();
+                $table->foreign('playlist_id', 'FK_playlists_items')->references('id')->on(config('database.connections.dbp_users.database') . '.user_playlists')->onDelete('cascade')->onUpdate('cascade');
+                $table->string('fileset_id', 16);
+                $table->foreign('fileset_id', 'FK_bible_filesets_plan_days_items')->references('id')->on(config('database.connections.dbp.database') . '.bible_filesets')->onUpdate('cascade')->onDelete('cascade');
+                $table->char('book_id', 3);
+                $table->foreign('book_id', 'FK_books_plan_days_items')->references('id')->on(config('database.connections.dbp.database') . '.books');
+                $table->integer('chapter_start')->unsigned();
+                $table->integer('chapter_end')->unsigned()->nullable();
+                $table->integer('verse_start')->unsigned();
+                $table->integer('verse_end')->unsigned()->nullable();
+                $table->integer('duration');
+                $table->timestamp('created_at')->default(DB::raw('CURRENT_TIMESTAMP'));
+                $table->timestamp('updated_at')->default(DB::raw('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'));
+                $table->integer('order_column');
+            });
+        }
+
```

Comment:
plan days may optionally include links to bonus external content, that doesn't seem included here.  perhaps a separate table is needed.  BTW, we would like to allow playlists to also optionally include an external link (this hasn't been discussed with design team yet) - i mention as another similarity between plan days and playlists.

Reply:
Done Jon, now the playlist has a new column `external_content`. This allows having external content on the playlists and each day of the plans